manifests/init.pp

```diff
@@ -19,6 +19,9 @@
 # $proxy_password::     Proxy password for authentication
 #                       type:String
 #
+# $pulp_max_speed::     Maximum bandwidth used per download in bytes/sec from CDN
```

Comment:
Small thing but I don't really like `CDN` here, as this will effect the speed of any RPM downloads.